bot/src/browser-actions.ts

```diff
@@ -298,15 +298,15 @@ export async function sendPromptUsingWs(
       channel_id: threadId,
       session_id: token,
       data: {
-        version: "1166847114203123795", // command version?
+        version: "11237876415471554623", // command version?
```

Comment:
@hgezim you have an extra `1` at the beginning of this. Why don't just merge my PR?? this seems exactly the same changes.